build-logic/binary-compatibility/src/test/kotlin/gradlebuild/binarycompatibility/SinceAndIncubatingRulesKotlinTest.kt

```diff
@@ -114,6 +114,7 @@ class SinceAndIncubatingRulesKotlinTest : AbstractBinaryCompatibilityTest() {
     """
 
     @Test
+    @Suppress("LongMethod")
```

Comment:
❓ What about we relax rules such as `LongMethod`, `TooManyFunctions` or `LargeClass` for test sources instead?

Reply:
I'm making this change. Works nicely for `LongMethod` and `LargeClass`, but I'm just going to leave `TooManyFunctions` disabled everywhere, because most cases are in prod code and the value it's adding seems dubious to me (comment if you don't agree).